scripts/generate/mockgen.go

```diff
@@ -50,29 +50,20 @@ func main() {
 	packageName := os.Args[1]
 	interfaces := os.Args[2]
 	outputPath := os.Args[3]
-	re := regexp.MustCompile("(?m)[\r\n]+^.*" + projectVendor + ".*$")
-
+	re := regexp.MustCompile(projectVendor)
 	copyrightHeader := fmt.Sprintf(copyrightHeaderFormat, time.Now().Year())
 
 	path, _ := filepath.Split(outputPath)
 	err := os.MkdirAll(path, os.ModeDir|0755)
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
-	}
+	handleError(err)
```

Comment:
Yeah.  Moving code into other functions and having real returns also lets normal things like `defer` work properly, which is useful.